README.md

```diff
@@ -74,17 +72,25 @@ All of the cli commands defined in this file are executed before the server is s
 execute offline commands otherwise the container will fail. For example, including `create cache ...` in the batch would
 fail as it requires a connection to an Infinispan server.
 
-Below is an example Identities batch CLI file `identities.batch`, that defines two users and their role:
+Infinispan provides implicit roles for some users.
+
+[TIP] Check Infinispan [documentation](https://infinispan.org/docs/stable/titles/configuring/configuring.html#default-user-roles_security-authorization)
+to know more about implicit roles and authorization
+
+Below is an example Identities batch CLI file `identities.batch`, that defines four users and their role:
 
 ```bash
 user create "Alan Shearer" -p "striker9" -g admin
-user create "Nolberto Solano" -p "winger7" -g dev
+user create "observer" -p "secret1" 
+user create "deployer" -p "secret2" 
+user create "Rigoberta Baldini" -p "secret3" -g monitor
 ```
 
-To run the image using a local `idenitities.batch`, execute:
+To run the image using a local `identities.batch`, execute:
 
 ```bash
-docker run -v $(pwd):/user-config -e IDENTITIES_BATCH="/user-config/identities.batch" infinispan/server
+docker volume create user-config 
```

Comment:
This line isn't necessary, the `-v` flag with `docker run ...` automatically creates the volume.

Reply:
it was not working for me, but maybe for anothe reason I will recheck